compiler/rustc_passes/src/reachable.rs

```diff
@@ -378,6 +378,8 @@ impl<'a, 'tcx> ItemLikeVisitor<'tcx> for CollectPrivateImplItemsVisitor<'a, 'tcx
     fn visit_impl_item(&mut self, _impl_item: &hir::ImplItem<'_>) {
         // processed in visit_item above
     }
+
+    fn visit_foreign_item(&mut self, _foreign_item: &hir::ForeignItem<'_>) {}
```

Comment:
```suggestion
    fn visit_foreign_item(&mut self, _foreign_item: &hir::ForeignItem<'_>) {
        // We never export foreign functions as they have no body to export.
    }
```
At least that's how I understand this, not completely sure about that :laughing: